packages/service-library/src/servicelib/long_running_tasks/_errors.py

```diff
@@ -46,8 +46,3 @@ class GenericClientError(BaseLongRunningError):
     msg_template: str = (
         "Unexpected error while '{action}' for '{task_id}': status={status} body={body}"
     )
-
-
-class TaskClientResultError(BaseLongRunningError):
```

Comment:
- unused? 
- Q: What are you using the code for? 
- NOTE that OsparcErrorMixin automatially produces the codes from inhertance tree.